### DISCUSSION TITLE
Decorator should also be able to return JSON or Dictionary object

### DISCUSSION THREAD

**Discussion by nathan-vo810** [langfuse/langfuse#1830] [Ideas]
### Describe the feature or potential improvement

Take the following snippet as an example

```
@observe()
def fn():
    return {"output": "something"}
```

Currently, when using the observe decorator for function, the output is returned in a string format. Thus, in the trace, when the return value may be a dictionary it is enclosed with double quote: `"{"output": "something"}"`

It would be great if we can return the value in the form of a JSON or dictionary like it behaves when using callbacks in Langchain.
```
{
    "output": "something"
}
```

**Comment by marcklingen** [comment]
hi @nathan-vo810, totally agree on this one. we made some improvements to the serialization logic to serialize more complex objects/classes and this is a regression. turning this into an issue

**Comment by hassiebp** [comment]
Hi @nathan-vo810 - a fix has been [merged to main just now](https://github.com/langfuse/langfuse-python/commit/2a0868c6bf14c341c6a8712be0e174aa692e89d2) and will be published in our next release. Thanks for your report!
